mempool/src/shared_mempool/coordinator.rs

```diff
@@ -44,7 +44,7 @@ use vm_validator::vm_validator::TransactionValidation;
 pub(crate) async fn coordinator<V>(
     mut smp: SharedMempool<V>,
     executor: Handle,
-    network_events: Vec<(NodeNetworkId, MempoolNetworkEvents)>,
+    network_events: Vec<(NetworkId, MempoolNetworkEvents)>,
```

Comment:
we should create a new `MempoolNetworkEvents` that does the select_all and enriches the events with the corresponding `PeerNetworkId`. Maybe in a follow up? One thing at a time : p

Reply:
Yeah, I'm trying to contain this to "mostly" removing `NodeNetworkId`, and I made `PeerNetworkId` not a tuple (because tuples are hard to read!)